pixi.toml

```
@@ -305,6 +305,8 @@ py-build = "pixi run -e py py-build-common"
 # Dedicated alias for building the python bindings in release mode for the `py` environment.
 py-build-release = "pixi run -e py py-build-common-release"
 
+check-py-signatures = "python scripts/ci/check_python_signatures.py"
```

To be consistent with the rest, and more easily findable through autocomplete/tab spam
```suggestion
py-check-signatures = "python scripts/ci/check_python_signatures.py"